substrate/frame/society/src/migrations.rs

```diff
@@ -20,10 +20,9 @@
 use super::*;
 use alloc::{vec, vec::Vec};
 use codec::{Decode, Encode};
-use frame_support::traits::{Defensive, DefensiveOption, Instance, UncheckedOnRuntimeUpgrade};
-
-#[cfg(feature = "try-runtime")]
-use sp_runtime::TryRuntimeError;
+use frame::try_runtime::TryRuntimeError;
```

Comment:
shouldn't this be behind the feature?
```suggestion
#[cfg(feature = "try-runtime")]
use frame::try_runtime::TryRuntimeError;
```